benchmark/single-source/SetTests.swift

```diff
@@ -135,10 +211,133 @@ public let SetTests = [
+
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Empty.Int",
+    runFunction: { n in run_SetIsSupersetSeqInt(setAB, arrayE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int.Empty",
+    runFunction: { n in run_SetIsSupersetSeqInt(setE, arrayAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int0",
+    runFunction: { n in run_SetIsSupersetSeqInt(setCD, arrayAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setCD, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Box0",
+    runFunction: { n in run_SetIsSupersetSeqBox(setOCD, arrayOAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOCD, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int25",
+    runFunction: { n in run_SetIsSupersetSeqInt(setAB, arrayB, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Box25",
+    runFunction: { n in run_SetIsSupersetSeqBox(setOAB, arrayOB, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int50",
+    runFunction: { n in run_SetIsSupersetSeqInt(setXY, arrayY, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setXY, arrayY]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Seq.Int100",
+    runFunction: { n in run_SetIsSupersetSeqInt(setP, arrayP, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setP, arrayP]) }),
+
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Seq.Empty.Int",
+    runFunction: { n in run_SetIsStrictSupersetSeqInt(setAB, arrayE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Seq.Int.Empty",
+    runFunction: { n in run_SetIsStrictSupersetSeqInt(setE, arrayAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Seq.Int0",
+    runFunction: { n in run_SetIsStrictSupersetSeqInt(setCD, arrayAB, false, 500 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setCD, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Seq.Box0",
+    runFunction: { n in run_SetIsStrictSupersetSeqBox(setOCD, arrayOAB, false, 500 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOCD, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Seq.Int25",
+    runFunction: { n in run_SetIsStrictSupersetSeqInt(setAB, arrayB, true, 50 * n) },
```

Comment:
```suggestion
    runFunction: { n in run_SetIsStrictSupersetSeqInt(setB, arrayBC, false, 50 * n) },
```
* Analogously for `Box25`.
* Use `setY` and `arrayYZ` on `Int50` below.
* Adjust `setUpFunction`s to match.

Reply:
https://github.com/apple/swift/pull/24156#discussion_r277914847
I think keep it true is better. https://github.com/apple/swift/pull/24156#discussion_r277914678

Reply:
Karoy was commenting on a Set.Set variant, back there:
>   runFunction: { n in run_SetIsStrictSupersetInt(setB, setAB, false, 50 * n) },

 which would have otherwise [short-circuited on the count conditions](https://github.com/apple/swift/pull/21300/files#diff-c8ef6275ed4edb2d48e444ed6c67dbdbR1186) because `setB.count < setAB.count`.

You have to flip the result here, because B is not a super set of AB. We are trying to exercise the [early exit on line 606](https://github.com/apple/swift/pull/21300/files#diff-5e43cf325f04c0738a77b7b8eb412e51R606).